### DISCUSSION TITLE
forms, params in rails controllers and sequel

### DISCUSSION THREAD

**Discussion by benkoshy** [jeremyevans/sequel#1866] [General]
Hi all,

- I was wanting to try out sequel in lieu of active record. I ripped out active record from my latest rails 7 project. 
- I gather that I could use `Sequel::Model.plugin :active_model` in combination with rails form helpers which is helpful. Or I suppose I could use forme. Anything else i'm missing.

But I have a few questions:

- On form submission, how would I handle: (i) strong params, and (ii) saving those params to a record in a rails controller?

Any pointers would be much appreciated. chrs!

**Comment by jeremyevans** [comment]
"Strong params" is a Rails issue, and I assume you can handle params the same way you would when using ActiveRecord.  However, I would avoid the use of "strong params", and use Sequel methods such as `Model#set_fields`:

```ruby
model = Model.new # or Model.with_pk!(model_id) for existing record
model.set_fields(params['model'], %w'field1 field2 ...')
model.save
```

**Reply by benkoshy** [reply]
Hi thanks for the response. 

I will try `set_fields`, and also have a go at handling params as with active record.
